### PR TITLE
EWPP-629: Fix select none.

### PR DESCRIPTION
**Pull request by drishu**
### Description

Fix bug where selecting value none would trigger error.

### Change log

- Changed: MediaCreationForm media_bundle config.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ vendor
 yarn.lock
 phpunit.xml
 docker-compose.*.yml
+/.gitattributes
```
Comment (upchuk) [inline .gitignore:15]: why this change?

**Review by upchuk — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (drishu) [reply .gitignore:15]: I've tested, it seems this gets added by phpstorm (it happened automatically when pulling a branch).

**Review by upchuk — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]
